### PR TITLE
RLP-979: update with develop

### PR DESCRIPTION
**Pull request by mortelli**
This PR updates the `rif-comms` branch with the latest changes from `develop`, per issue [RLP-979](https://jirainfuy.atlassian.net/browse/RLP-979).

## pending
- [x] `pytest` results
- [x] approvals

### CODE REVIEW

File: raiden/api/rest.py [hunk 1]
```diff
@@ -20,13 +20,10 @@
...
 from webargs.flaskparser import parser
 
-from raiden.api.objects import (
-    AddressList,
-    PartnersPerTokenList,
-    DashboardGeneralItem,
-    DashboardGraphItem,
-    DashboardTableItem
-)
+from raiden.api.objects import AddressList, PartnersPerTokenList
```
Comment (mortelli) [inline raiden/api/rest.py:23]: @JONAF2103 would you be so kind to check the changes here? i believe they stem from one of your PRs and i'm not so sure about their correctness

thanks

**Review by marcosmartinez7 — APPROVED** [review]
This LGTM but i think that @JONAF2103 opinion is more valid here

File: raiden/api/rest.py (hunk 1 above)
Comment (marcosmartinez7) [reply raiden/api/rest.py:23]: is this comment related with the imports? 

seems just like a reorg right?

Comment (JONAF2103) [reply raiden/api/rest.py:23]: Sure

**Review by JONAF2103 — APPROVED** [review]
LGTM

File: raiden/api/rest.py (hunk 1 above)
Comment (mortelli) [reply raiden/api/rest.py:23]: > is this comment related with the imports?
> 
> seems just like a reorg right?

i meant in the entire module; i just picked the topmost change to make the comment for the file :stuck_out_tongue: 

sorry i wasn't clear enough

**Comment by mortelli** [comment]
pytest results look ok (uploaded [here](https://github.com/anarancio/lumino-docs/tree/master/testing/pytest/results/rif-comms/2020-12-31))

**Pull request merged**